tests/baselines/reference/moduleResolutionWithExtensions_unexpected.trace.json

```diff
@@ -2,11 +2,8 @@
     "======== Resolving module 'normalize.css' from '/a.ts'. ========",
     "Module resolution kind is not specified, using 'NodeJs'.",
     "Loading module 'normalize.css' from 'node_modules' folder, target file type 'TypeScript'.",
-    "'package.json' does not have a 'typings' field.",
-    "'package.json' does not have a 'types' field.",
```

Comment:
why do these lines go away?

Reply:
Because we don’t look at the field to calculate sub module names

Reply:
Previously when sub module name was empty string these fields were read to get better name